RELEASE/scripts/autoscend/quests/level_07.ash

```diff
@@ -109,7 +109,7 @@ boolean L7_crypt()
 	// delay crypt for cold medicine cabinet usage unless we have run out of other stuff to do
 	// crypt is underground so med cab will generate breathitins, 5 turns free outside
 	// todo: currently assumes CMC will get used. Wait needlessly if we don't end up using CMC
-	if(auto_is_valid($item[cold medicine cabinet]) && auto_haveColdMedCabinet() && 
+	if(auto_is_valid($item[cold medicine cabinet]) && item_amount($item[cold medicine cabinet]) > 0 && 
 		get_workshed() != $item[cold medicine cabinet] && !isAboutToPowerlevel())
```

Comment:
Maybe add an additional check in the if statement for the new auto_workshed to equal cold medicine cabinet/cmc or auto or LX_getDesiredWorkshed() where it is equal to either $item[cold medicine cabinet] or $item[none]. Because you are already checking that the person has it but it currently isn't installed, this would verify that the user intends for it to be installed